src/main/features/claimant-response/views/check-and-send.njk

```diff
@@ -11,47 +11,89 @@
 
     {{ tableStart('yourResponse', 'Your response') }}
 
-    {% if draft.settleAdmitted %}
-      {% if draft.settleAdmitted.admitted.option === 'yes' %}
-        {% set settleAdmittedResponse = 'I accept this amount' %}
-      {% else %}
-        {% set settleAdmittedResponse = 'I reject this amount' %}
+    {% if alreadyPaid %}
+
+      {% if draft.partPaymentReceived %}
+        {% set paymentReceivedResponse = 'Yes' if draft.partPaymentReceived.received.option === 'yes' else 'No' %}
```

Comment:
please use `YesNoViewFilter` for these. e.g. `draft.partPaymentReceived.received | renderYesNo`

Reply:
YesNoViewFilter takes boolean input, using it here would result in it always rendering 'Yes'

Reply:
Create new filter for `YesNoOption` and use that. You may do that in separate PR if you like as this is being held for too long.